platform/openide.util/test/unit/src/org/openide/util/BaseUtilitiesTest.java

```diff
@@ -100,6 +103,7 @@ public void testWhatIsFreeBSD () {
         assertFalse ("freebsd is not isWindows", BaseUtilities.isWindows ());
         assertTrue ("freebsd isUnix", BaseUtilities.isUnix ());
     }
+    
```

Comment:
Please avoid changing formatting as this creates noise

Reply:
reverted in https://github.com/apache/netbeans/pull/5115/commits/19b692f079b0d19cac28093333066891ff68477d